graphics/jxrlib/Portfile

```
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                jxrlib
+version             1.2_git20170615-f752187
+set version_name    1.2~git20170615.f752187
```

You can use a bit of Tcl here so as not to have to hardcode and update this every time:

```suggestion
set version_name    [string map {- .} ${version}]
```

---

Thanks done with `regsub`.